docs/log-levels.md

```diff
@@ -14,7 +14,8 @@ SwiftLog defines the following 7 log levels via the [`Logger.Level` enum](https:
 * `error`
 * `critical`
 
-Out of those, only levels _less severe than_ info (exclusive) are generally okay to be used by libraries.
+Out of those, only levels _less severe than_ info (exclusively) are generally okay to be used by libraries.
```

Comment:
This is a good general overview, but does not define the alternative (returning error object, throwing exception) and does not refer to error handling document.

Also, in situations where there's a complex result i.e. applying an invalid configuration to an engine, this doesn't talk about [accumulation](https://www.scalactic.org/user_guide/OrAndEvery) of potential warnings and errors.